addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -137,11 +137,27 @@ function align(editor, mode) {
 function colorElement(element, color, mode) {
     const newClassName = element.className
         .replace(mode === 'color' ? TEXT_CLASSES_REGEX : BG_CLASSES_REGEX, '')
+        .replace(' gradient_text', '')
```

Comment:
gradient_text seems like a strange name for a class (at least text_gradient? and is not there a convention for classes in the editor lib? I don't know).

What is the class used for?

Reply:
Text classes appear to be named "text-", which makes them mutually exclusive when uses on a text chunk, which is what is wanted here. => renamed the class `text-gradient`.
Its purpose is mostly to apply/remove the styles related to making the `background-image` gradient visible through the writing. Secondarily it makes it possible to recognize a foreground gradient (when the class is set) from an highlight gradient (when the class is not set).